base/socket.jl

```diff
@@ -312,6 +312,10 @@ mutable struct TCPServer <: LibuvServer
         return tcp
     end
 end
+
+# Keyword arg "delay": if true, libuv delays creation of socket fd till bind.
+# It can be set to false if there is a need to set socket options before
+# further calls to `bind` and `listen`, e.g. `SO_REUSEPORT`.
```

Comment:
I meant as part of the doc string for the `TCPServer` constructor, although a comment is better than nothing.

Reply:
TCPServer is not exported, so I think this should be okay for now.